examples/joystick/Joystick.cpp

```diff
@@ -142,15 +142,15 @@ int main()
         object.value.setString("N/A");
     }
 
-    for (Texts::iterator it = texts.begin(); it != texts.end(); ++it)
+    for (auto& [k, v] : texts)
```

Comment:
Maybe change the names to `[label, joystick]` ("k/v" is not really more expressive than "first/second").

Also below.

Reply:
Ditto, will change.